packages/app/src/sandbox/eval/transpilers/babel/worker/evaluate.js

```diff
@@ -1,13 +1,16 @@
 import resolve from 'browser-resolve';
```

Comment:
I had to rewrite some parts in the babel evaluator. It turns out that we didn't cache according to node spec! We do now, which makes the babel vue preset work.